packetnetworking/utils.py

```diff
@@ -202,6 +202,24 @@ def generate_persistent_names_udev():
     return {"etc/udev/rules.d/70-persistent-net.rules": persistent_udev}
 
 
+def generate_persistent_names_mdev():
+    mdevconf = """\
+        # This file was automatically generated by the equinix metal installation environment.
+
+        -SUBSYSTEM=net;DEVPATH=.*/net/.*;.*     root:root 600 @/sbin/nameif -s
+    """
+
+    mactab = """\
+        # This file was automatically generated by the equinix metal installation environment.
```

Comment:
can we get this as Equinix Metal instead, please

Reply:
i fixed this in a later commit